pkg/core/logging/default.go

```diff
@@ -0,0 +1,29 @@
+package logging
+
+var defaultPatterns = []MaskPattern{
+	// JWT
+	NewRegexMaskPattern(`eyJ[A-Za-z0-9-_=]+\.[A-Za-z0-9-_=]+\.?[A-Za-z0-9-_.+/=]*`),
+	// Session Tokens
+	NewRegexMaskPattern(`[A-Fa-f0-9-]{36}\.[A-Za-z0-9]+`),
+}
+
+func NewDefaultMaskedTextFormatter(sensitiveStrings []string) *MaskedTextFormatter {
+	patterns := defaultPatterns
```

Comment:
I guess `defaultPatterns` is being modified. We should write `patterns := defaultPattern[:]` to copy the slice first.

Reply:
I don't think it is modified? `MaskedTextFormatter` should not modify the input patterns, and if sensitive strings are provided a slice is created using `append`. However, I'll make a defensive copy just in case.